data/events/2024/fortaleza/main.yml

```
@@ -0,0 +1,249 @@
+event_twitter: "devopsdayce" # Change this to the twitter handle for your event such as DevOpsDayschi or DevOpsDaysmsp
+description: "DevOpsDays is coming to Fortaleza!" # Edit this to suit your preferences
+#ga_tracking_id: "G-V6NPDB4RM8" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+gtm_tracking_id: "G-V6NPDB4RM8" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+
+# All dates are in unquoted 2023-MM-DDTHH:MM:SS+TZ:TZ, like this:
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow 2023-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate:  2024-09-21T08:00:00-03:00  # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate:  2024-09-21T17:00:00-03:00  # The end date of your event. Leave blank if you don't have a venue reserved yet.
+
+# Leave CFP dates blank if you don't know yet, or set all three at once.
+cfp_date_start: 2024-04-26T10:00:00-03:00 # start accepting talk proposals.
+cfp_date_end: 2024-05-26T07:00:00-03:00 # close your call for proposals.
+cfp_date_announce: 2024-07-04T08:00:00-03:00 # inform proposers of status
+
+cfp_open: "true"
+cfp_link: "https://www.papercall.io/devopsdaysfortaleza2024" #if you have a custom link for submitting proposals, add it here. This will control the Propose menu item as well as the "Propose" button.
+
+# registration_date_start: 2023-06-29 # start accepting registration. Leave blank if registration is not open yet. This will make the "Register" button appear on your "Welcome" page.
+# registration_date_end: 2023-08-10 # close registration. Leave blank if registration is not open yet. If you set "registration_date_start" you need a value here.
+
+registration_closed: "" #set this to true if you need to manually close registration before your registration end date
+#registration_link: "https://www.eventbrite.com.br/e/devops-days-fortaleza-tickets-638733989557" # If you have a custom registration link, enter it here. This will control the Registration menu item as well as the "Register" button.
+
+# Location
+#
+coordinates: "-3.764104, -38.4869504" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
+location: "Local: Estácio - Universidade Estácio do Ceará - Fortaleza/CE" # Defaults to city, but you can make it the venue name.
+#
+location_address: "R. Eliseu Uchôa Beco, 600 - Guararapes" #Optional - use the street address of your venue. This will show up on the welcome page if set.
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+  #- name: propose
+  - name: location
+  # - name: registration
+  #   url: https://devopsdaysce.eventize.com.br
+  - name: program
+  - name: speakers
```

Strongly suggest disabling this link until you actually have speakers to share.
```suggestion
  - name: speakers
```

---

@phrawzty done.
Can you check, please?